### PR TITLE
SDKS-2279_Deleting WebAuthn Credentials

### PR DESCRIPTION
**Pull request by george-bafaloukas-forgerock**
Exposing method to clear the credentials for WebAuthn from the WebAuthnRegistration callback and the FRUser class

# JIRA Ticket

[SDKS-2279](https://bugster.forgerock.org/jira/browse/SDKS-2279)

### CODE REVIEW

File: FRAuth/FRAuth/User/FRUser.swift [hunk 1]
```diff
@@ -376,7 +376,6 @@ public class FRUser: NSObject, NSSecureCoding {
         return BrowserBuilder(oAuth2Client, frAuth.keychainManager)
     }
     
-
```
Comment (rodrigoareis) [inline FRAuth/FRAuth/User/FRUser.swift:379]: If this is an intentional change, please update copyright

File: FRAuth/FRAuth/WebAuthn/Authenticator/Internal/CredentialSource.swift [hunk 2]
```diff
@@ -10,7 +10,7 @@
 import Foundation
 import FRCore
 
-struct PublicKeyCredentialSource {
```
Comment (rodrigoareis) [inline FRAuth/FRAuth/WebAuthn/Authenticator/Internal/CredentialSource.swift:13]: Update copyright. Once it's now public, should we add doc?

File: FRAuth/FRAuth/WebAuthn/Authenticator/PlatformAuthenticator/PlatformAuthenticator.swift [hunk 3]
```diff
@@ -108,4 +108,19 @@ class PlatformAuthenticator: Authenticator {
         FRLog.v("Starting getAssertionSession", subModule: WebAuthn.module)
         return PlatformAuthenticatorGetAssertionSession(config: self.config, keySupportChooser: self.keySupportChooser, credentialsStore: self.credentialsStore, authenticatorDelegate: self.authenticationDelegate)
     }
+
```
Comment (rodrigoareis) [inline FRAuth/FRAuth/WebAuthn/Authenticator/PlatformAuthenticator/PlatformAuthenticator.swift:111]: Update copyright

File: FRAuth/FRAuth/Callbacks/WebAuthnRegistrationCallback.swift [hunk 4]
```diff
@@ -57,6 +57,7 @@ open class WebAuthnRegistrationCallback: WebAuthnCallback {
     /// Boolean indicator whether or not Callback response is AM 7.1.0 or above
     var isNewJSONFormat: Bool = false
     var pubCredAlg: [COSEAlgorithmIdentifier] = []
+    var platformAuthenticator: PlatformAuthenticator?
```
Comment (rodrigoareis) [inline FRAuth/FRAuth/Callbacks/WebAuthnRegistrationCallback.swift:60]: Update copyright

File: FRAuth/FRAuth/WebAuthn/FRWebAuthn.swift [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+import FRCore
+
+public class FRWebAuthn: NSObject {
```
Comment (rodrigoareis) [inline FRAuth/FRAuth/WebAuthn/FRWebAuthn.swift:15]: Missing doc?

**Review by rodrigoareis — CHANGES_REQUESTED** [review]
Overall changes looks good. Left some comments to be addressed

File: FRAuth/FRAuth/WebAuthn/FRWebAuthn.swift (hunk 5 above)
Comment (jeyanthanperiyasamy) [reply FRAuth/FRAuth/WebAuthn/FRWebAuthn.swift:15]: yeah we need a doc here , since its publicly exposed

**Review by vahancouver — APPROVED** [review]
Changes look good to me

**Review by jeyanthanperiyasamy — APPROVED** [review]
approved

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

**Review by george-bafaloukas-forgerock — COMMENTED** [review]
looks good Stoyan

**Review by spetrov — APPROVED** [review]
👍🏻

**Pull request merged**